_posts/2017-06-01-how-to-build-dotnet-core-project-with-vs-code.md

```diff
@@ -0,0 +1,375 @@
+  Restoring packages for C:\Projects\dotnetcore-vscode\dotnetcorevscode.Library\dotnetcorevscode.Library.csproj...
+  Restoring packages for C:\Projects\dotnetcore-vscode\dotnetcorevscode.Tests\dotnetcorevscode.Tests.csproj...
+  Generating MSBuild file C:\Projects\dotnetcore-vscode\dotnetcorevscode.Library\obj\dotnetcorevscode.Library.csproj.nuget.g.props.
+  Generating MSBuild file C:\Projects\dotnetcore-vscode\dotnetcorevscode.Library\obj\dotnetcorevscode.Library.csproj.nuget.g.targets.
+  Writing lock file to disk. Path: C:\Projects\dotnetcore-vscode\dotnetcorevscode.Library\obj\project.assets.json
+  Restore completed in 867,29 ms for C:\Projects\dotnetcore-vscode\dotnetcorevscode.Library\dotnetcorevscode.Library.csproj.
+  Generating MSBuild file C:\Projects\dotnetcore-vscode\dotnetcorevscode.Tests\obj\dotnetcorevscode.Tests.csproj.nuget.g.props.
+  Generating MSBuild file C:\Projects\dotnetcore-vscode\dotnetcorevscode.Tests\obj\dotnetcorevscode.Tests.csproj.nuget.g.targets.
+  Writing lock file to disk. Path: C:\Projects\dotnetcore-vscode\dotnetcorevscode.Tests\obj\project.assets.json
+  Restore completed in 1,72 sec for C:\Projects\dotnetcore-vscode\dotnetcorevscode.Tests\dotnetcorevscode.Tests.csproj.
+  Generating MSBuild file C:\Projects\dotnetcore-vscode\dotnetcorevscode.Web\obj\dotnetcorevscode.Web.csproj.nuget.g.props.
+  Generating MSBuild file C:\Projects\dotnetcore-vscode\dotnetcorevscode.Web\obj\dotnetcorevscode.Web.csproj.nuget.g.targets.
+  Writing lock file to disk. Path: C:\Projects\dotnetcore-vscode\dotnetcorevscode.Web\obj\project.assets.json
+  Restore completed in 2,41 sec for C:\Projects\dotnetcore-vscode\dotnetcorevscode.Web\dotnetcorevscode.Web.csproj.
+
+  NuGet Config files used:
+      C:\Users\joffr\AppData\Roaming\NuGet\NuGet.Config
+      C:\Program Files (x86)\NuGet\Config\Microsoft.VisualStudio.Offline.config
+
+  Feeds used:
+      https://api.nuget.org/v3/index.json
+      C:\Program Files (x86)\Microsoft SDKs\NuGetPackages\
+
+joffr@JOFFREY-LAPTOP MINGW64 /c/Projects/dotnetcore-vscode
+$ dotnet build
+Microsoft (R) Build Engine version 15.1.1012.6693
+Copyright (C) Microsoft Corporation. All rights reserved.
+
+  dotnetcorevscode.Library -> C:\Projects\dotnetcore-vscode\dotnetcorevscode.Library\bin\Debug\netstandard1.4\dotnetcorevscode.Library.dll
+  dotnetcorevscode.Tests -> C:\Projects\dotnetcore-vscode\dotnetcorevscode.Tests\bin\Debug\netcoreapp1.1\dotnetcorevscode.Tests.dll
+  dotnetcorevscode.Web -> C:\Projects\dotnetcore-vscode\dotnetcorevscode.Web\bin\Debug\netcoreapp1.1\dotnetcorevscode.Web.dll
+
+Build succeeded.
+    0 Warning(s)
+    0 Error(s)
+
+Time Elapsed 00:00:05.96
+```
+
+### <a name='Addprojectsreferencesoneachprojects'></a>Add projects references on each projects
+
+On Visual Studio 2017, if you create a new project, you have to add a reference on it by using ``Right click on a project > Add > Reference > ...`` And on our solution, I have to add few references : 
```

Comment:
du coup, ici "add ... to" :)

Et est-ce que ce n'est pas aussi "a few"? j'hésite dans le contexte... Alexandre? :)